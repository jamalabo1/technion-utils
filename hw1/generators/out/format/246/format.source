D W 908 1
B Z 775 2
A Z 167 3
C W 284 4
B Y 107 5
C X 267 6
B W 769 7
D X 613 8
D Y 45 9
D W 12 10
C W 591 11
A W 165 12
C X 843 13
D W 930 14
A Y 439 15
B Z 70 16
B X 430 17
A Z 683 18
B Z 420 19
D W 960 20
B X 418 21
D Y 746 22
C Z 150 23
D Z 898 24
C Z 901 25
A W 824 26
D Z 749 27
D Y 38 28
D W 510 29
D X 285 30
D X 901 31
D X 318 32
A Z 904 33
D X 464 34
B W 421 35
B W 251 36
B X 21 37
D X 694 38
D W 536 39
A W 465 40
D W 958 41
B X 261 42
D Y 725 43
D Y 284 44
B Z 471 45
B W 950 46
A X 965 47
B X 85 48
B X 139 49
B Z 229 50
B Z 352 51
B W 661 52
C Z 901 53
D Y 245 54
D Z 116 55
A Y 503 56
A W 375 57
A X 262 58
D X 900 59
B Z 847 60
D Y 680 61
A Z 900 62
D X 646 63
C W 164 64
A Y 719 65
C Z 873 66
A Z 351 67
B Z 802 68
C X 411 69
B X 704 70
D Z 691 71
A W 222 72
C Y 14 73
D Z 364 74
B X 771 75
C W 558 76
C X 112 77
B Z 774 78
A Z 919 79
D X 57 80
C Z 290 81
C W 546 82
A X 687 83
A X 901 84
B W 518 85
A Y 252 86
A X 105 87
A X 821 88
B X 958 89
B W 687 90
C Y 268 91
D X 47 92
C W 416 93
B Z 554 94
B W 575 95
D Y 547 96
C Z 751 97
C X 355 98
A X 249 99
B Y 175 100
C Y 520 101
D Y 13 102
A Z 545 103
A W 767 104
D Y 19 105