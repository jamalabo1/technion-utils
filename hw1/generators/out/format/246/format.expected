Blockchain Info:
1
Sender Name: D
Receiver Name: Y
Transaction Value: 19
Transaction timestamp: 105
2
Sender Name: A
Receiver Name: W
Transaction Value: 767
Transaction timestamp: 104
3
Sender Name: A
Receiver Name: Z
Transaction Value: 545
Transaction timestamp: 103
4
Sender Name: D
Receiver Name: Y
Transaction Value: 13
Transaction timestamp: 102
5
Sender Name: C
Receiver Name: Y
Transaction Value: 520
Transaction timestamp: 101
6
Sender Name: B
Receiver Name: Y
Transaction Value: 175
Transaction timestamp: 100
7
Sender Name: A
Receiver Name: X
Transaction Value: 249
Transaction timestamp: 99
8
Sender Name: C
Receiver Name: X
Transaction Value: 355
Transaction timestamp: 98
9
Sender Name: C
Receiver Name: Z
Transaction Value: 751
Transaction timestamp: 97
10
Sender Name: D
Receiver Name: Y
Transaction Value: 547
Transaction timestamp: 96
11
Sender Name: B
Receiver Name: W
Transaction Value: 575
Transaction timestamp: 95
12
Sender Name: B
Receiver Name: Z
Transaction Value: 554
Transaction timestamp: 94
13
Sender Name: C
Receiver Name: W
Transaction Value: 416
Transaction timestamp: 93
14
Sender Name: D
Receiver Name: X
Transaction Value: 47
Transaction timestamp: 92
15
Sender Name: C
Receiver Name: Y
Transaction Value: 268
Transaction timestamp: 91
16
Sender Name: B
Receiver Name: W
Transaction Value: 687
Transaction timestamp: 90
17
Sender Name: B
Receiver Name: X
Transaction Value: 958
Transaction timestamp: 89
18
Sender Name: A
Receiver Name: X
Transaction Value: 821
Transaction timestamp: 88
19
Sender Name: A
Receiver Name: X
Transaction Value: 105
Transaction timestamp: 87
20
Sender Name: A
Receiver Name: Y
Transaction Value: 252
Transaction timestamp: 86
21
Sender Name: B
Receiver Name: W
Transaction Value: 518
Transaction timestamp: 85
22
Sender Name: A
Receiver Name: X
Transaction Value: 901
Transaction timestamp: 84
23
Sender Name: A
Receiver Name: X
Transaction Value: 687
Transaction timestamp: 83
24
Sender Name: C
Receiver Name: W
Transaction Value: 546
Transaction timestamp: 82
25
Sender Name: C
Receiver Name: Z
Transaction Value: 290
Transaction timestamp: 81
26
Sender Name: D
Receiver Name: X
Transaction Value: 57
Transaction timestamp: 80
27
Sender Name: A
Receiver Name: Z
Transaction Value: 919
Transaction timestamp: 79
28
Sender Name: B
Receiver Name: Z
Transaction Value: 774
Transaction timestamp: 78
29
Sender Name: C
Receiver Name: X
Transaction Value: 112
Transaction timestamp: 77
30
Sender Name: C
Receiver Name: W
Transaction Value: 558
Transaction timestamp: 76
31
Sender Name: B
Receiver Name: X
Transaction Value: 771
Transaction timestamp: 75
32
Sender Name: D
Receiver Name: Z
Transaction Value: 364
Transaction timestamp: 74
33
Sender Name: C
Receiver Name: Y
Transaction Value: 14
Transaction timestamp: 73
34
Sender Name: A
Receiver Name: W
Transaction Value: 222
Transaction timestamp: 72
35
Sender Name: D
Receiver Name: Z
Transaction Value: 691
Transaction timestamp: 71
36
Sender Name: B
Receiver Name: X
Transaction Value: 704
Transaction timestamp: 70
37
Sender Name: C
Receiver Name: X
Transaction Value: 411
Transaction timestamp: 69
38
Sender Name: B
Receiver Name: Z
Transaction Value: 802
Transaction timestamp: 68
39
Sender Name: A
Receiver Name: Z
Transaction Value: 351
Transaction timestamp: 67
40
Sender Name: C
Receiver Name: Z
Transaction Value: 873
Transaction timestamp: 66
41
Sender Name: A
Receiver Name: Y
Transaction Value: 719
Transaction timestamp: 65
42
Sender Name: C
Receiver Name: W
Transaction Value: 164
Transaction timestamp: 64
43
Sender Name: D
Receiver Name: X
Transaction Value: 646
Transaction timestamp: 63
44
Sender Name: A
Receiver Name: Z
Transaction Value: 900
Transaction timestamp: 62
45
Sender Name: D
Receiver Name: Y
Transaction Value: 680
Transaction timestamp: 61
46
Sender Name: B
Receiver Name: Z
Transaction Value: 847
Transaction timestamp: 60
47
Sender Name: D
Receiver Name: X
Transaction Value: 900
Transaction timestamp: 59
48
Sender Name: A
Receiver Name: X
Transaction Value: 262
Transaction timestamp: 58
49
Sender Name: A
Receiver Name: W
Transaction Value: 375
Transaction timestamp: 57
50
Sender Name: A
Receiver Name: Y
Transaction Value: 503
Transaction timestamp: 56
51
Sender Name: D
Receiver Name: Z
Transaction Value: 116
Transaction timestamp: 55
52
Sender Name: D
Receiver Name: Y
Transaction Value: 245
Transaction timestamp: 54
53
Sender Name: C
Receiver Name: Z
Transaction Value: 901
Transaction timestamp: 53
54
Sender Name: B
Receiver Name: W
Transaction Value: 661
Transaction timestamp: 52
55
Sender Name: B
Receiver Name: Z
Transaction Value: 352
Transaction timestamp: 51
56
Sender Name: B
Receiver Name: Z
Transaction Value: 229
Transaction timestamp: 50
57
Sender Name: B
Receiver Name: X
Transaction Value: 139
Transaction timestamp: 49
58
Sender Name: B
Receiver Name: X
Transaction Value: 85
Transaction timestamp: 48
59
Sender Name: A
Receiver Name: X
Transaction Value: 965
Transaction timestamp: 47
60
Sender Name: B
Receiver Name: W
Transaction Value: 950
Transaction timestamp: 46
61
Sender Name: B
Receiver Name: Z
Transaction Value: 471
Transaction timestamp: 45
62
Sender Name: D
Receiver Name: Y
Transaction Value: 284
Transaction timestamp: 44
63
Sender Name: D
Receiver Name: Y
Transaction Value: 725
Transaction timestamp: 43
64
Sender Name: B
Receiver Name: X
Transaction Value: 261
Transaction timestamp: 42
65
Sender Name: D
Receiver Name: W
Transaction Value: 958
Transaction timestamp: 41
66
Sender Name: A
Receiver Name: W
Transaction Value: 465
Transaction timestamp: 40
67
Sender Name: D
Receiver Name: W
Transaction Value: 536
Transaction timestamp: 39
68
Sender Name: D
Receiver Name: X
Transaction Value: 694
Transaction timestamp: 38
69
Sender Name: B
Receiver Name: X
Transaction Value: 21
Transaction timestamp: 37
70
Sender Name: B
Receiver Name: W
Transaction Value: 251
Transaction timestamp: 36
71
Sender Name: B
Receiver Name: W
Transaction Value: 421
Transaction timestamp: 35
72
Sender Name: D
Receiver Name: X
Transaction Value: 464
Transaction timestamp: 34
73
Sender Name: A
Receiver Name: Z
Transaction Value: 904
Transaction timestamp: 33
74
Sender Name: D
Receiver Name: X
Transaction Value: 318
Transaction timestamp: 32
75
Sender Name: D
Receiver Name: X
Transaction Value: 901
Transaction timestamp: 31
76
Sender Name: D
Receiver Name: X
Transaction Value: 285
Transaction timestamp: 30
77
Sender Name: D
Receiver Name: W
Transaction Value: 510
Transaction timestamp: 29
78
Sender Name: D
Receiver Name: Y
Transaction Value: 38
Transaction timestamp: 28
79
Sender Name: D
Receiver Name: Z
Transaction Value: 749
Transaction timestamp: 27
80
Sender Name: A
Receiver Name: W
Transaction Value: 824
Transaction timestamp: 26
81
Sender Name: C
Receiver Name: Z
Transaction Value: 901
Transaction timestamp: 25
82
Sender Name: D
Receiver Name: Z
Transaction Value: 898
Transaction timestamp: 24
83
Sender Name: C
Receiver Name: Z
Transaction Value: 150
Transaction timestamp: 23
84
Sender Name: D
Receiver Name: Y
Transaction Value: 746
Transaction timestamp: 22
85
Sender Name: B
Receiver Name: X
Transaction Value: 418
Transaction timestamp: 21
86
Sender Name: D
Receiver Name: W
Transaction Value: 960
Transaction timestamp: 20
87
Sender Name: B
Receiver Name: Z
Transaction Value: 420
Transaction timestamp: 19
88
Sender Name: A
Receiver Name: Z
Transaction Value: 683
Transaction timestamp: 18
89
Sender Name: B
Receiver Name: X
Transaction Value: 430
Transaction timestamp: 17
90
Sender Name: B
Receiver Name: Z
Transaction Value: 70
Transaction timestamp: 16
91
Sender Name: A
Receiver Name: Y
Transaction Value: 439
Transaction timestamp: 15
92
Sender Name: D
Receiver Name: W
Transaction Value: 930
Transaction timestamp: 14
93
Sender Name: C
Receiver Name: X
Transaction Value: 843
Transaction timestamp: 13
94
Sender Name: A
Receiver Name: W
Transaction Value: 165
Transaction timestamp: 12
95
Sender Name: C
Receiver Name: W
Transaction Value: 591
Transaction timestamp: 11
96
Sender Name: D
Receiver Name: W
Transaction Value: 12
Transaction timestamp: 10
97
Sender Name: D
Receiver Name: Y
Transaction Value: 45
Transaction timestamp: 9
98
Sender Name: D
Receiver Name: X
Transaction Value: 613
Transaction timestamp: 8
99
Sender Name: B
Receiver Name: W
Transaction Value: 769
Transaction timestamp: 7
100
Sender Name: C
Receiver Name: X
Transaction Value: 267
Transaction timestamp: 6
101
Sender Name: B
Receiver Name: Y
Transaction Value: 107
Transaction timestamp: 5
102
Sender Name: C
Receiver Name: W
Transaction Value: 284
Transaction timestamp: 4
103
Sender Name: A
Receiver Name: Z
Transaction Value: 167
Transaction timestamp: 3
104
Sender Name: B
Receiver Name: Z
Transaction Value: 775
Transaction timestamp: 2
105
Sender Name: D
Receiver Name: W
Transaction Value: 908
Transaction timestamp: 1